johnsonandjohnson/Bodiless-JS
# fix(vital-layouts): Make footer more flexible by separating our rewards/copyright

## Changes
- Separated out Rewards styling from Base
- Separated out CopyrightRow to make it more reusable, as well as made a static version

## Test Instructions
- Verify static & edit footer is identical on all breakpoints.
-- I ran visuals on homepage on 7 different sizes to confirm footer was doing the same thing -> 
![2022-06-16_10-53-55](https://user-images.githubusercontent.com/58534559/174125627-e1702659-40af-4304-bf13-fb193bbeb72f.jpg). (this is on my own applitools account so you can't access results)
- but for testing just bring up vital-demo site and should see no difference between local and https://main-bvxea6i-vadijmmtl4ax4.us-4.platformsh.site/
